ui/de-lib/src/test/java/org/iplantc/de/client/services/impl/MessageServiceFacadeImplTest.java

```diff
@@ -0,0 +1,94 @@
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+
+/**
+ * @author aramsey
+ */
+@RunWith(GwtMockitoTestRunner.class)
+public class MessageServiceFacadeImplTest {
+
+    @Mock NotificationAutoBeanFactory notesFactoryMock;
+    @Mock DEProperties dePropertiesMock;
+    @Mock DiscEnvApiService deServiceFacadeMock;
+    @Mock UserInfo userInfoMock;
+    @Mock DiskResourceUtil diskResourceUtilMock;
+    @Mock AsyncCallback<String> asyncCallbackMock;
+
+    private MessageServiceFacadeImpl uut;
+
+    @Before
+    public void setUp() {
+
+        when(dePropertiesMock.getMuleServiceBaseUrl()).thenReturn("sampleUrl");
+        uut = new MessageServiceFacadeImpl(deServiceFacadeMock,
+                                           dePropertiesMock,
+                                           notesFactoryMock,
+                                           userInfoMock);
+    }
+
+    @Test
+    public void testDeleteAll_noFilter() {
+        String filter = "ALL";
+        uut.deleteAll(filter, asyncCallbackMock);
+        assertEquals(filter.toLowerCase(), NotificationCategory.ALL.toString().toLowerCase());
+    }
+
+    @Test
+    public void testDeleteAll_filterData() {
+        String filter = "DATA";
+        uut.deleteAll(filter, asyncCallbackMock);
+        assertNotEquals(filter.toLowerCase(), NotificationCategory.ALL.toString().toLowerCase());
+        assertEquals(filter.toLowerCase(), NotificationCategory.DATA.toString().toLowerCase());
+    }
+
+    @Test
+    public void testDeleteAll_filterToolRequests() {
+        String filter = "TOOL REQUEST";
+        uut.deleteAll(filter, asyncCallbackMock);
+        assertNotEquals(filter.toLowerCase(), NotificationCategory.ALL.toString().toLowerCase());
+        assertEquals(filter.toLowerCase(), NotificationCategory.TOOLREQUEST.toString().toLowerCase());
+    }
+
+    @Test
+    public void testDeleteAll_filterApps() {
+        String filter = "APPS";
+        uut.deleteAll(filter, asyncCallbackMock);
+        assertNotEquals(filter.toLowerCase(), NotificationCategory.ALL.toString().toLowerCase());
+        assertEquals(filter.toLowerCase(), NotificationCategory.APPS.toString().toLowerCase());
+    }
+
+    @Test
+    public void testDeleteAll_filterAnalysis() {
+        String filter = "ANALYSIS";
+        uut.deleteAll(filter, asyncCallbackMock);
+        assertNotEquals(filter.toLowerCase(), NotificationCategory.ALL.toString().toLowerCase());
+        assertEquals(filter.toLowerCase(), NotificationCategory.ANALYSIS.toString().toLowerCase());
+    }
+
+    @Test
+    public void testDeleteAll_filterPermID() {
+        String filter = "PERMANENT ID REQUEST";
+        uut.deleteAll(filter, asyncCallbackMock);
+        assertNotEquals(filter.toLowerCase(), NotificationCategory.ALL.toString().toLowerCase());
+        assertEquals(filter.toLowerCase(), NotificationCategory.PERMANENTIDREQUEST.toString().toLowerCase());
```

Comment:
I might be missing something with these tests.
It seems like a `filter` string is created, then it's just compared to the `NotificationCategory` enums as strings?
I see the `filter` is passed to a MessageServiceFacadeImpl#deleteAll call, but it doesn't look like any of the facade's functionality is tested.
Unless we're capturing or verifying calls into the `deServiceFacadeMock`, maybe we don't need these tests?